clients/da-vinci-client/src/main/java/com/linkedin/davinci/helix/LeaderFollowerPartitionStateModel.java

```diff
@@ -136,8 +134,9 @@ public void onBecomeLeaderFromStandby(Message message, NotificationContext conte
      * where a slice doesn't have a replicating leader.  While this state transition executes, there should be no
      * other leader in the slice.
      */
-    updateLagMonitor(
+    heartbeatMonitoringService.updateLagMonitor(
         message.getResourceName(),
+        getPartition(),
         heartbeatMonitoringService::addLeaderLagMonitor,
```

Comment:
I might want to internalize this method. - Introduce a ENUM to represent the 3 actions